niworkflows/info.py

```diff
@@ -49,12 +49,12 @@
     'Programming Language :: Python :: 3.5',
 ]
 
-REQUIRES = ['nipype==8ddc', 'future', 'nilearn', 'sklearn', 'pandas', 'matplotlib',
+REQUIRES = ['nipype', 'future', 'nilearn', 'sklearn', 'pandas', 'matplotlib',
             'jinja2', 'svgutils', 'tinycss']
 SETUP_REQUIRES = []
 REQUIRES += SETUP_REQUIRES
 
-LINKS_REQUIRES = ['git+https://github.com/nipy/nipype.git@8ddca5a03fcad26887c862dc23c82ef23f2ee506#egg=nipype-8ddc',]
+LINKS_REQUIRES = [,]
```

Comment:
It's here: https://github.com/poldracklab/niworkflows/pull/56/files#diff-3254677a7917c6c01f55212f86c57fbfR51